src/App/utils/resourcesByType/index.js

```diff
@@ -0,0 +1,876 @@
+import React from 'react'
+import {internet, lorem, random, name} from 'faker'
+
+import Anchor, {types as anchorTypes} from 'package/components/Anchor'
+import Avatar, {sizes as avatarSizes} from 'package/components/Avatar'
+import Button, {sizes as buttonSizes} from 'package/components/Button'
+import Card, {levels as cardLevels} from 'package/components/Card'
+import ContainerWidth from 'package/components/ContainerWidth'
+import Error from 'package/components/Error'
+import HeaderCard from 'package/components/HeaderCard'
+import Heading, {levels as headingLevels} from 'package/components/Heading'
+import Icon, {types as iconTypes, sizes as iconSizes} from 'package/components/Icon'
+import IconLabel from 'package/components/IconLabel'
+import Image from 'package/components/Image'
+import InstructorLessons from 'package/components/InstructorLessons'
+import InstructorRevenue from 'package/components/InstructorRevenue'
+import InstructorStats from 'package/components/InstructorStats'
+import LayoutColumns from 'package/components/LayoutColumns'
+import LessonActions from 'package/components/LessonActions'
+import LessonOverviews from 'package/components/LessonOverviews'
+import LessonOverviewsByGroup from 'package/components/LessonOverviewsByGroup'
+import List from 'package/components/List'
+import Loading from 'package/components/Loading'
+import Markdown from 'package/components/Markdown'
+import Maybe from 'package/components/Maybe'
+import Open from 'package/components/Open'
+import Paragraph, {types as paragraphTypes} from 'package/components/Paragraph'
+import Prompt from 'package/components/Prompt'
+import Request, {methods as requestMethods} from 'package/components/Request'
+import RequestedLessons from 'package/components/RequestedLessons'
+import Tabs from 'package/components/Tabs'
+import Toggle, {selectedItems as toggleSelectedItems} from 'package/components/Toggle'
+
+import InstructorDashboard from 'package/screens/InstructorDashboard'
+import InstructorDetails from 'package/screens/InstructorDetails'
+import InstructorsDirectory from 'package/screens/InstructorsDirectory'
+import LessonDetails from 'package/screens/LessonDetails'
+import LessonsDirectory from 'package/screens/LessonsDirectory'
+import NewLesson from 'package/screens/NewLesson'
+
+import colors from 'package/utils/colors'
+import colorValues from 'package/utils/colorValues'
+import {containerWidths} from 'package/utils/hardCodedSizes'
+import {lessonStates} from 'package/utils/lessonStates'
+
+import Authentication from 'App/components/Authentication'
+
+const getLoginUrl = () => (
+  `${process.env.REACT_APP_EGGHEAD_BASE_URL}/users/jwt?return_to=${window.location.href}`
+)
+
+const createNodeExample = () => random.arrayElement([
```

Comment:
Each value uses faker where it makes sense  so that when you hit "Randomize" on the examples it re-generates valid input so you can test and develop with a variety of use cases.